code/game/objects/structures/watercloset.dm

```diff
@@ -414,6 +414,16 @@
 		user << "<span class='warning'>Someone's already washing here!</span>"
 		return
 
+	if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/monkeycube))
+		if (/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped)

```

Comment:
I didn't even know checking a typepath compiled (it really shouldn't). And yeah, handling this on an istype() in sinks is bad, afterattack is better encapsulation.